packages/clay-navigation/src/index.tsx

```diff
@@ -6,13 +6,25 @@
 
 import React from 'react';
 import {Breadcrumb} from './Breadcrumb';
+import {Nav} from './Nav';
+import {NavItem} from './NavItem';
+import {NavLink} from './NavLink';
+import {Vertical} from './Vertical';
 
 const ClayNavigation: React.FunctionComponent<any> & {
+	Nav: typeof Nav;
+	NavItem: typeof NavItem;
+	NavLink: typeof NavLink;
 	Breadcrumb: typeof Breadcrumb;
+	Vertical: typeof Vertical;
 } = ({children}) => {
 	return children;
 };
 
 ClayNavigation.Breadcrumb = Breadcrumb;
+ClayNavigation.Nav = Nav;
+ClayNavigation.NavItem = NavItem;
+ClayNavigation.NavLink = NavLink;
+ClayNavigation.Vertical = Vertical;
```

Comment:
I was thinking of making it available the same way we did in `ClayForm`. It seems a little strange... what do you think?

```javascript
ClayVertical.Nav = Nav;
ClayVertical.NavItem = NavItem;
ClayVertical.NavLink = NavLink;

export {ClayVertical};
```

Now I've been wondering if all the components should be in the same namespace, maybe you get the impression that it's all part of `Navigation`, `Form` and you can compose them all anywhere, it looks like we can sell more...

Reply:
I didn't want to namespace it off of `ClayVertical` because `nav` is used outside of the context of `Vertical` in other types of navs.

We could possible change it from `@clayui/navigation` to `@clayui/nav` and the way we can do something like

```js
ClayNav = Nav;
ClayNav.Item = NavItem;
ClayNav.Link = NavLink;


// option #1
ClayNav.Vertical = Vertical;
ClayNav.BreadCumb = BreadCumb;

// option #2
export {
       ClayNavWithBreadCrumb,
       ClayNavWithVertical,
}

export default ClayNav
```

Reply:
Hmm, makes sense. I think we can keep this within `Navigation` even though we may also migrate `Navigation Bar` to it.

Reply:
Right on, so ship components like so? This means our root offering for Navigation is `'nav'`. 

```js
ClayNavigation = Nav;
ClayNavigation.Item = NavItem;
ClayNavigation.Link = NavLink;
ClayNavigation.Vertical = Vertical;
ClayNavigation.BreadCumb = BreadCumb;

export default ClayNavigation
```

This makes sense to me I think.

Reply:
Hmm, maybe so. Not sure about this because the breadcrumb base is not a nav but I think we can go with that.